docs/reference/config.rst

```diff
@@ -1,39 +1,35 @@
 Configuration
 =============
 
-Beets has an extensive configuration system that lets you customize nearly
-every aspect of its operation. To configure beets, you create a file called
-``config.yaml``. The location of the file depend on your platform (type ``beet
-config -p`` to see the path on your system):
-
-* On Unix-like OSes, write ``~/.config/beets/config.yaml``.
-* On Windows, use ``%APPDATA%\beets\config.yaml``. This is usually in a
-  directory like ``C:\Users\You\AppData\Roaming``.
-* On OS X, you can use either the Unix location or ``~/Library/Application
+Beets has an extensive configuration system which lets you customize nearly
+every aspect of beets. To configure beets, a configuration file called
+``config.yaml`` must be created. The location of ``config.yaml`` depends on your platform. You can type ``beet config -p`` to see where ``config.yaml`` is on your system.
+
+* On Unix-like OSes, use ``~/.config/beets/config.yaml``.
+* On Windows, use ``%APPDATA%\beets\config.yaml``. ``%APPDATA%`` is usually an environment variable
+  which stores a directory like ``C:\Users\[Your Windows' username]\AppData\Roaming``.
+* On OS X, you can use either the Unix OS's location or ``~/Library/Application
   Support/beets/config.yaml``.
 
-You can launch your text editor to create or update your configuration by
-typing ``beet config -e``. (See the :ref:`config-cmd` command for details.) It
-is also possible to customize the location of the configuration file and even
-use multiple layers of configuration. See `Configuration Location`_, below.
+You can launch your text editor to create or update your ``config.yaml`` by
+typing ``beet config -e`` (See the :ref:`config-cmd` command for details.). `Configuration Location`_ describes how you can customize ``config.yaml``'s location.
 
-The config file uses `YAML`_ syntax. You can use the full power of YAML, but
-most configuration options are simple key/value pairs. This means your config
-file will look like this::
+``config.yaml`` uses `YAML`_ syntax. Most of beets' configuration options are 
+key/value pairs which look like these examples::
```

Comment:
Any particular reason to use “like these examples” instead of “like this”? It’s wordier, and I think it’s already clear that these are examples.